content-team/working-docs/social-media-policy.md

```diff
@@ -0,0 +1,90 @@
+# Social Media Strategy and Best Practices
+DD&I Content Team
+
+## Channels
+- Twitter
+- Instagram
+- Drupaldiversity.com blog
+- Newsletter 
+
+## Purpose
+*What are we attempting to achieve with our social media?*
+
+- To cultivate a more inclusive Drupal community.
+- To give recognition to the diverse voices we have.
+- Transparency, public participation and collaboration
+
+## Audience
+*Who are we trying to reach?*
+
+- For the community.
+- For the volunteers who contribute to our issue queue.
+- For mentors, and teachers in teh Drupal community.
+- For people who organize and attend Drupal events.
+
+## Content
+*What types of content do we post? How does that content support our mission? What do our audiences want to hear?*
+
+We want to post content that is related to diversity (age/race/gender/religion/more) and/or tech and various places where these issues cross over. Sharing content with different perspectives and difference voices allows us to learn and grow in our understanding.
+
+We want people in the community to feel included. By being more aware of how we treat each other and being intentional about the kinds of things we post, we can demonstrate that awareness.
+
+I also think we should look through the list of articles we’ve been compiling to read and see what should be shared, and make sure we are sharing factual resources as well - various information on diversity 101, links to sessions, presentations, articles, faqs, etc.  
+
+## Tone
+*What is the tone of our communication? What types of interactions are encouraged/discouraged?*
+
+- We assume good intentions until proven otherwise.
+- Rational
+- Professional
+- Knowledgeable
+- Compassionate
+- Considerate
+- Educational
+
+On social media, we will strive to encourage positive interactions with followers and be a compassionate voice in the community.
+ We will:
+- Be careful of what we like or retweet.  Like things judiciously and that apply directly to our mission. Do not “like” anything we would not tweet from the account.
+
+In the face of negative interactions, we will consider replying with compassion versus ignoring i.e:
+  - “We’d love to have you involved, how can we help?”
+  - “We’re sorry you feel that way, can we better explain?”
+
+The content team will be used as a resource to discuss whether a negative interaction should be replied to and how. Serious situations will be escalated to the Leadership team for advice.
+
+
+## Personal accountability
+*Do we disclose who is tweeting? Is that important if there's a unified message?*
```

Comment:
@AlannaBurke Hard to say for sure. I'll confirm with the leadership team and team leads. So far, no one has objected to it :) but hopefully putting the draft up during the meeting will help with that.